bin/bs_ios

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# bs_ios uploads app binaries for UI testing on BrowserStack.
+#
+# This version of the script works only with the V2 endpoint (XC Test Plans).
+# This script assumes that Xcode Test Plan name is TestPlan.xctestplan.
```

Comment:
If so, let's put a custom error message when Test Plan name is not "TestPlan.xctestplan"

Reply:
done